### DISCUSSION TITLE
Repo Images suddenly stopped displaying 10-15-24.

### DISCUSSION THREAD

**Discussion by suzannepk** [community/community#141682] [Programming Help]
### Select Topic Area

Question

### Body

The images in our repo suddenly stopped displaying yesterday. We did not commit any thing that would change them and I have tried refreshing my browser cache and checking for extraneous "/". When I go to the folder where we store the images in the repo I cannot even download the image files. 
for example see https://github.com/olcf/hands-on-with-frontier/blob/master/images/frontier_logo_2023.png.

This is the repo: https://github.com/olcf/hands-on-with-frontier. 

Is there a general problem with images and Github at the moment or is this a problem with our specific repo?

**Comment by davevad93** [comment]
Hi @suzannepk ,

I've checked the repository and I can view and download the images just fine.

<img width="1253" alt="1" src="https://github.com/user-attachments/assets/1da63281-df05-4601-834f-cc1aa51168c4">
<br>
<br>

So the problem is neither with GitHub nor with your specific repo, it must be something related to your browser or the network, try using another browser and check if you can display them correctly.
